packages/core/src/utils/password.ts

```diff
@@ -90,6 +98,23 @@ export const executeLegacyHash = async (
   const resolvedArgs = args.map((arg) => (arg === '@' ? inputPassword : arg));
   const inputString = resolvedArgs.join('');
 
+  if (isPbkdf2Algorithm(algorithm)) {
+    const [salt, iterations = '10_000', keylen = '64', digest = 'sha512'] = resolvedArgs;
```

Comment:
@wangsijie please double check if the default values are reasonable and well-balanced in terms of security and resource consumption

Reply:
That value is just an example from the Node.js documentation, but I think we can remove the default since "Legacy" implies the data was migrated from another source, so all parameters should already be known.